autoload/fern/renderer/nerdfont.vim

```diff
@@ -19,16 +19,54 @@ endfunction
 function! s:render(nodes) abort
   let options = {
         \ 'leading': g:fern#renderer#nerdfont#leading,
+        \ 'padding': g:fern#renderer#nerdfont#padding,
+        \ 'markers': g:fern#renderer#nerdfont#markers,
+        \ 'root_symbol': g:fern#renderer#nerdfont#root_symbol,
         \}
   let base = len(a:nodes[0].__key)
+
+  if options.markers
+    let length_nodes = len(a:nodes)
+    let levels = {}
+
+    for i in range(length_nodes - 1, 0, -1)
+      let node = a:nodes[i]
+      let node.level = len(node.__key) - base
```

Comment:
Attributes without `_` prefix are not opened (See `:help fern-develop-node`). Please add a prefix like `_renderer_nerdfont_` so that other plugin won't overwrite the attribute like

```vim
let node._renderer_nerdfont_level = len(node.__key) - base
```

Or put all required attributes in a dict like

```vim
let params = {}
let params.level = len(node.__key) - base
" ...
let node._renderer_nerdfont = params
```